content/applications/marketing/events/event_templates.rst

```diff
@@ -0,0 +1,214 @@
+  option, and allows visitors to create community rooms of their own.
+- :guilabel:`Register Button`: adds a button at the end of the event submenu that takes visitors to
+  the event-specific registration page when clicked.
+
+Once the desired checkboxes have been ticked, select an approrpriate :guilabel:`Timezone` for the
+event from the available drop-down menu.
+
+Then, for organizational purposes, users have the option to add :guilabel:`Tags` to this event
+template.
+
+There is also the option to :guilabel:`Limit Registrations` to this specific event template by
+ticking that checkbox. If ticked, proceed to enter the number of :guilabel:`Attendees` this template
+should be limited to.
+
+Beneath those general information fields at the top of the event template form, there are five tabs:
+
+- :ref:`Tickets <create_events/event_form_tickets>`
+- :ref:`Communication <create_events/event_form/communication>`
+- :ref:`Booths <event_templates/event_template_form/booths>`
+- :ref:`Questions <create_events/event_form/questions>`
+- :ref:`Notes <create_events/event_form/notes>`
+
+.. _event_templates/event_template_form/booths:
+
+Booths tab
+~~~~~~~~~~
+
+The :guilabel:`Booths` tab on an event template form is the only tab that differentiates itself from
+a standard event form, where the other tabs (:guilabel:`Tickets`, :guilabel:`Communication`,
+:guilabel:`Questions`, and :guilabel:`Notes`) are present and configured using the same process.
+
+.. important::
+   In order for a booth or booth category to be created, an event booth product **must** be created
+   in the database first, with the *Product Type* set to *Event Booth*. **Only** products with that
+   specific configuration can be selected in the required *Product* field of a booth or booth
+   category form.
+
+.. note::
+   Event booths can be created and customized in two ways in the Odoo *Events* application. Either
+   in the :guilabel:`Booths` tab of an event template form, or by navigating to
+   :menuselection:`Events app --> Configuration --> Booth Categories`.
+
+To add a booth from the :guilabel:`Booths` tab of an event template form, click :guilabel:`Add a
+line`. Doing so reveals a blank :guilabel:`Create Booths` pop-up window.
+
+.. image:: event_templates/create-booths-popup.png
+   :align: center
+   :alt: A Create Booths pop-up window in the Odoo Events application.
+
+Start by providing a :guilabel:`Name` for this booth in the corresponding field at the top of the
+pop-up window.
+
+Then, select an appropriate :guilabel:`Booth Category` from the drop-down field below. Booth
+categories can be created and modified from the *Booth Categories* page in the *Events* application,
+which is accessible by navigating to :menuselection:`Events app --> Configuration --> Booth
+Categories`.
+
+.. tip::
+   A :guilabel:`Booth Category` can be created directly from this field on the :guilabel:`Create
```

Comment:
I was thinking about the info in the tip admonition – it's quite lengthy but seems really important. Maybe we could take it out of the tip block since it's not exactly extra info, and a "tip" usually implies something extra.

But since the tip covers various ways of creating a Booth category, I was thinking we could give "Booth Categories" its own dedicated subsection, maybe with an h5 tag (*****). What do you think?

Reply:
I opted for an admonition here, because once the brief tangent about Booth Categories is complete, I want the reader to know (both visually and procedurally) that they can pick up the next step process -- I feel like admonitions, like this, provide some much need "boundaries" for instances like this. That said, I agree, its on the "lengthier" side of things when it comes to admonition blocks -- but I think it still works. However, if it gets brought up once again in a review, I'll make some necessary adjustments, and again, give you full credit for pointing it out first 👍